cmd/kubelet/app/server.go

```diff
@@ -457,7 +457,7 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 
 	if kubeDeps.CAdvisorInterface == nil {
 		kubeDeps.CAdvisorInterface, err = cadvisor.New(uint(s.CAdvisorPort), s.ContainerRuntime, s.RootDirectory)
-		if err != nil {
+		if err != nil && goruntime.GOOS != "darwin" {
```

Comment:
@liggitt thanks for looking into this! That is exactly what I thought and wanted to hear. Do you think the community would be ok with me making such a platform specific interface? it seems like darwin more and more is become a oversight in development. I dont want to dedicate time to something that the community will not be happy to have.